container/lxd/initialisation_linux.go

```diff
@@ -123,19 +123,20 @@ func getConfigSetterConnect() (configSetter, error) {
 }
 
 type configSetter interface {
```

Comment:
probably the name of this interface needs to change, since it isn't just setting config anymore.
lxdServer ?